### PR TITLE
agent-creation

### PR DESCRIPTION
**Pull request by nviolante25**
# Agent

Solves #5 . 

This PR depends on PR #4 

- [ ] Add Agent class to interact with the environment from PR #4

### CODE REVIEW

**Review by emasquil — CHANGES_REQUESTED** [review]
Please rename the directory to `ppo` and after merging #4 rebase this and we can then merge it

File: ppo/agent/agent.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import acme
+import chex
+from dm_env import specs
+import jax
+from jax.random import PRNGKey
+import numpy as np
+
+from env_wrapper.pendulum_wrapper import PendulumEnv
```
Comment (theovincent) [inline ppo/agent/agent.py:8]: I think it should be:
```
from ppo.env_wrapper.pendulum_wrapper import PendulumEnv
```

File: ppo/agent/agent.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+from networks import PolicyNetwork
```
Comment (theovincent) [inline ppo/agent/agent.py:9]: I think it should be:
```
from ppo.networks import PolicyNetwork
```

**Review by theovincent — CHANGES_REQUESTED** [review]

**Comment by emasquil** [comment]
this became outdated respect to the new PR #12

**Pull request closed without merging**